src/squawk/SquawkAssignment.cpp

```diff
@@ -173,8 +189,15 @@ namespace UKControllerPlugin {
             const EuroScopeCRadarTargetInterface & radarTarget
         ) const
         {
-
-            if (!this->activeCallsigns.UserHasCallsign()) {
+            /*
+             * We have to check for all zero's because of how ES handles flightplan events
+             * halfway across Europe.
+             */
+            if (

```

Comment:
Can these be put into a single function, perhaps? There are 3 very similar bits of code :D

Reply:
dc165378